src/routes/cards_generation.py

```diff
@@ -137,6 +137,14 @@ def drawMetaname(draw: ImageDraw.ImageDraw, metaname: str, color: RGBAColor) ->
     drawMetaname(draw, card_metadata.card_metaname, card_metadata.text_meta_color)
 
     log.debug(f"    Card contents: {lyrics}")
+
+    for lyric_line in lyrics:
```

Comment:
ce for existe déjà quelques lignes plus bas, inclues-y ta logique au lieu de dupliquer